x-pack/plugins/actions/server/authorization/get_authorization_mode_by_source.ts

```diff
@@ -32,3 +34,40 @@ export async function getAuthorizationModeBySource(
     ? AuthorizationMode.Legacy
     : AuthorizationMode.RBAC;
 }
+
+export async function getBulkAuthorizationModeBySource(
+  unsecuredSavedObjectsClient: SavedObjectsClientContract,
+  executionOptions: ExecuteOptions[] = []
```

Comment:
nit: passing in an array of ExecutionSources would make this function more generic and align better with `getAuthorizationModeBySource`. However, it doesn't look like this is used outside of the actions plugin so it's not a big deal.

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137781/commits/554740872b25aab2bbdd695e51d97fbcbd1aec9c)